include/crow/http_server.h

```diff
@@ -107,6 +108,7 @@ namespace crow
                         {
                             try
                             {
+                                task_queue_length_pool_[i] = 0;
```

Comment:
Are you sure this won't randomly reset the `task_queue_length`s when they shouldn't be?

Reply:
No I'm not sure. At first I thought it was executed only when no task was left, but it seems that I misread the code.. Corrected :)